src/ImageSharp/Memory/NullMemoryManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace SixLabors.ImageSharp.Memory
+{
+    /// <summary>
+    /// Implements <see cref="MemoryManager"/> by allocating new buffers on every call.
+    /// </summary>
+    public class NullMemoryManager : MemoryManager
```

Comment:
I think we need to find a better name for this :) `NullMemoryManager` suggests me, that it's always returning `null` instead of a memory block with a valid memory address.

@JimBobSquarePants @tocsoft suggestions?

Reply:
`SimpleMemoryManager` maybe?

Reply:
What about ManagedMemoryFactory and PooledMemoryFactory ?